testing/TestHarness/TestHarness.Shared/BaseHostInitialization.cs

```diff
@@ -67,11 +67,11 @@ protected virtual IHostBuilder Configuration(IHostBuilder builder)
 			// Only use this syntax for UI tests - use UseConfiguration in apps
 			builder = builder.ConfigureAppConfiguration((ctx, b) =>
 			{
-				b.AddEmbeddedConfigurationFile<App>(file);
+				b.AddEmbeddedConfiguration<App>(ctx, file);
```

Comment:
Why was this changed?

Reply:
I did that to reflect commit 0e8240c71f922ea78678261f61d4ac9b2d8eae99 which marked `AddEmbeddedConfigurationFile<T>` `private`. That method appeared to have been exposed by mistake, but we did invoke it from test harness. The change you noticed above updated the test to reference only a `public` method.

Reply:
Done!